content/en/blog/_posts/2022-12-12-host-process-containers-stable/index.md

```diff
@@ -0,0 +1,113 @@
+development pipeline.
+
+## How does it work?
+
+Windows HostProcess containers are implemented with Windows _Job Objects_, a break from the
+previous container model using server silos. Job objects are components of the Windows OS which offer the ability to
+manage a group of processes as a group (a.k.a. _jobs_) and assign resource constraints to the
+group as a whole. Job objects are specific to the Windows OS and are not associated with the Kubernetes [Job API](https://kubernetes.io/docs/concepts/workloads/controllers/job/). They have no process or file system isolation,
+enabling the privileged payload to view and edit the host file system with the
+correct permissions, among other host resources. The init process, and any processes
+it launches or that are explicitly launched by the user, are all assigned to the
+job object of that container. When the init process exits or is signaled to exit,
+all the processes in the job will be signaled to exit, the job handle will be
+closed and the storage will be unmounted.
+
+HostProcess and Linux privileged containers enable similar scenarios but differ
+greatly in their implementation (hence the naming difference). HostProcess containers
+have their own pod security policies. Those used to configure Linux privileged
+containers **do not** apply. Enabling privileged access to a Windows host is a
+fundamentally different process than with Linux so the configuration and
+capabilities of each differ significantly. Below is a diagram detailing the
+overall architecture of Windows HostProcess containers:
+
+{{< figure src="hpc-architecture.png" alt="HostProcess Architecture" >}}
+
+Two major features were added prior to moving to stable: the ability to run as local user accounts, and
+a simplified method of accessing volume mounts. For more info visit our [updated documentation page](/docs/tasks/configure-pod-container/create-hostprocess-pod)
+
+## Use Cases
+
+The SIG-Windows community has been busy putting HostProcess containers to use - even before GA! CThey've
+been very excited to use HostProcess containers for a number of important activities that were a pain to
+perform in the past. Now using HostProcess containers you can install the windows-exporter on your nodes
+and collect extremely detailed monitoring information with Prometheus. You can now run csi-proxy, flannel,
+calico, event loggers, and more as HPCs.
+
+To see more HPC examples head over to [HostProcess Examples](https://github.com/kubernetes-sigs/sig-windows-tools/tree/master/hostprocess)
+
+## How do I use it?
+
+A HostProcess container can be built using any base image of your choosing, however, for convenience we have
+created [a HostProcess container base image](https://github.com/microsoft/windows-host-process-containers-base-image).
+This image is only a few KB in size and does not inherit any of the same compatibility requirements as regular Windows
+server containers.
+
+All you need in your Dockerfile is the following URI:
```

Comment:
```suggestion
To use that Microsoft image, put this in your `Dockerfile`:
```
`mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v1.0.0` is not a URI, despite the insistence of Docker, Inc.

Reply:
According to https://en.wikipedia.org/wiki/Uniform_Resource_Identifier I think this still qualifies as a URI.
It is not a URL tho

> A Uniform Resource Identifier (URI) is a unique sequence of characters that identifies a logical or physical resource used by web technologies. URIs may be used to identify anything, including real-world objects, such as people and places, concepts, or information resources such as web pages and books. Some URIs provide a means of locating and retrieving information resources on a network (either on the Internet or on another private network, such as a computer filesystem or an [Intranet](https://en.wikipedia.org/wiki/Intranet)); these are [Uniform Resource Locators](https://en.wikipedia.org/wiki/Uniform_Resource_Locator) (URLs). A URL provides the location of the resource.